### PR TITLE
Add Apple TV jailbreaks guide, switch UntetherHomeDepot repo (mine has the community offsets preinstalling), fix a grammar issue

### PR DESCRIPTION
**Pull request by WhitetailAni**
- Added blackb0x guide
- Added ChimeraTV guide
- Added ElectraTV guide
- Added backr00m guide
- Added LiberTV guide
- Fixed a grammar mistake - its, not it's in the line "Sideloadly must make a request to **it's** servers in order to work with free developer accounts. If you are not OK with this, you may use an alternate Apple ID.
- Changed the repo listed for JailbreakMe 4.0 and Home Depot to my repo as it has an updated untether with the community offsets (before, you had to get them manually which was annoying). Meant to PR this awhile ago but just remembered now

I'm not sure how to add an Apple TV section to the homepage or embed images - my blackb0x guide has a spot for an aluminum remote to be shown but I'm not sure how to add it.
Hopefully you guys do!

### CODE REVIEW

**Review by TheMasterOfMike — APPROVED** [review]
Pretty much looks good here, thanks.

Couple minor things:
- Down the road (doesn’t have to be in this PR, can be done later) how feasible would it be to make a checkra1n for tvOS guide?
- I’m going to have Emma look this over before it’s merged because there’s probably other changes that need to be implemented on the guide (and maybe AppleDB) in order for Apple TV’s to be listed properly.

**Comment by RealKGB** [comment]
I can add a checkra1n section to tvOS. I also want to add a Patchstick guide and more stuff for ATVs but that requires getting ATVs on the necessary firmware versions - and since no one really cares about the ATV, getting them on TV Software 4 and early 5/tvOS 9/10/11 is difficult.

Also, how do you properly link images?

**Comment by RealKGB** [comment]
Alright, added checkra1n ATV guides. Also fixed up the delayOTA section - 15.7.1 and 16.1.x all expired, plus 15.6RC1 was unsigned

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - /ra1n
+  - /installing-odysseyrain
+  - /odysseyra1n
+  - /odysseyrain
+  - /or
+  - /installing-checkra1n
+  - /installing-checkrain
+  - /checkra1n
+  - /checkrain
+  - /cr
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/index.md:17]: this section should be modified as it will conflict with the existing checkra1n page

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+lang: en_US
+title: Installing Odysseyra1n (ChromeOS)
+description: Guide to installing Odysseyra1n on ChromeOS
+permalink: /installing-odysseyra1n-chromeos
+redirect_from:
+  - /installing-checkra1n/chromeos
+  - /installing-checkra1n-chromeos
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md:8]: Links and texts should be changed to reflect that it's a TV guide

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+::: warning
+
+If you are using a computer with an AMD Ryzen CPU, you will likely run into issues. If you do run into issues, you should use a Mac or a computer with an Intel CPU to follow this guide.
+
+:::
+
+::: warning
+
+If you have an Apple TV HD and are using a USB-C to UsB-C to do this process, you may run into issues entering into DFU mode
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md:26]: should be USB-C not UsB-C

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+::: danger
+
+This is a highly experimental guide page, there may be things not directly mentioned that need to be done in order to follow this. As a result, we highly recommend getting another type of computer instead if possible.
+
+:::
+
+::: tip
+
+You need to have enabled developer mode on your ChromeOS device to follow this
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md:40]: Should be a period at the end of this sentence, needs fixing on original chromeOS guide too

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+:::
+
+## Requirements (ChromeOS)
+
+- The latest release of [checkra1n](https://checkra.in)
+- An Apple TV Siri remote (any generation)
+- If you have an Apple TV 4K, you will also need two cables
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md:48]: Should be a colon at the end of this point

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+  - A GoldenEye, or Foxlink X892 cable
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md:49]: No comma necessary here

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+![A screenshot of the checkra1n application](/assets/images/checkra1n.png)
+
+## Allowing checkra1n to run
+
+1. Move the checkra1n binary which is downloaded to `/usr/local/bin`
+    - This can be done by doing the following:
+      - press `Ctrl` + `Alt` + `T`
+      - run `shell`
+      - run `mv ~/Downloads/checkra1n /usr/local/bin/`
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md:61]: Please capitalise the first letters of each point. This should be changed on original chromeos guide if it's there too

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+
+## Installing checkra1n
+
+1. Run the `checkra1n` binary in the terminal using `TERM=linux sudo checkra1n`
+    - You may need to run `sudo chmod a+x /usr/local/bin/checkra1n` if the binary doesn't run
+    - If you're on tvOS 14.6 or later, you will need to head into `Options`, then enable the `Allow Untested Versions` option once it runs
+1. Click `Start` and follow all onscreen prompts
+1. You will now be presented with instructions in how to reboot your device into <router-link to="/faq/#what-is-dfu-mode">DFU mode</router-link>, click `Start` to begin
+    - Follow the instructions until your device shows a black screen
+1. After this, checkra1n should automatically install.
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md:74]: Periods shouldn't be at the end of points

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-linux.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+::: warning
+
+If you are using a computer with an AMD Ryzen CPU, you will likely run into issues. If you do run into issues, you should use a Mac or a computer with an Intel CPU to follow this guide.
+
+:::
+
+::: warning
+
+If you have an Apple TV HD and are using a USB-C to UsB-C to do this process, you may run into issues entering into DFU mode
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-linux.md:27]: Should be USB-C, not UsB-C

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-linux.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+If you do have issues, get a USB-A to USB-C cable and, if necessary, also get a USB-C to USB-A adapter.
+
+:::
+
+## Requirements (Linux)
+
+- The latest release of [checkra1n](https://checkra.in)
+- An Apple TV Siri remote (any generation)
+- If you have an Apple TV 4K, you will also need two cables
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-linux.md:37]: Should be a colon at the end of this

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-linux.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+:::
+
+1. Run the `checkra1n` binary in the terminal using `./checkra1n`
+    - You may need to run `sudo chmod a+x ./checkra1n` if the binary doesn't run
+    - If you're on tvOS 14.6 or later, you will need to head into `Options`, then enable the `Allow Untested Versions` option once it runs
+1. Click `Start` and follow all onscreen prompts
+1. You will now be presented with instructions in how to reboot your device into <router-link to="/faq/#what-is-dfu-mode">DFU mode</router-link>, click `Start` to begin
+    - Follow the instructions until your device shows a black screen
+1. After this, checkra1n should automatically install.
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-linux.md:60]: Should be no period at the end of points

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+  - TheHacker894
+  - zachary7829
+  - WhitetailAni
+---
+
+## Requirements
+
+- The latest release of [checkra1n](https://checkra.in)
+- An Apple TV Siri remote (any generation)
+- If you have an Apple TV 4K, you will also need two cables
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md:18]: Should be a colon at the end of this

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+  - A GoldenEye, or Foxlink X892 cable
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md:19]: No comma required here

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
+---
+lang: en_US
+title: Installing checkra1n for Apple TVs (macOS)
+description: Guide to installing checkra1n for Apple TVs on macOS
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md:4]: should be Apple TV, not Apple TVs

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+
+![A screenshot of the checkra1n application](/assets/images/checkra1n.png)
+
+## Installing checkra1n
+
+:::
+
+::: warning
+
+If you have an Apple TV HD and are using a USB-C to UsB-C to do this process, you may run into issues entering into DFU mode
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md:31]: should be USB-C not UsB-C

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+
+If you do have issues, get a USB-A to USB-C cable and, if necessary, also get a USB-C to USB-A adapter.
+
+:::
+
+1. Open checkra1n on your computer
+1. Plug your Apple TV device into your computer
+1. If you are using an Apple TV 4K (1st generation), connect your GoldenEye to your Apple TV's LAN port, and then connect your DCSD to the GoldenEye's Lightning port. Connect the USB-A end of the DCSD to your computer.
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md:39]: This should be separated into multiple points as there is too much info here

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+    - If you're on tvOS 14.6 or later, you will need to head into `Options`, then enable the `Allow Untested Versions` option once it runs
+1. Click `Start` -> `Next` on checkra1n
+    - Your device will be put into recovery mode automatically
+1. You will now be presented with instructions in how to reboot your device into <router-link to="/faq/#what-is-dfu-mode">DFU mode</router-link>, click `Start` to begin
+    - Follow the instructions until your device shows a black screen
+1. After this, checkra1n should automatically install.
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-macos.md:45]: Should not be a period at the end of this

File: docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md [hunk 18]
```diff
@@ -0,0 +1,112 @@
...
+  - /odysseyn1x
+  - /odysseynix
+  - /using-bootra1n
+  - /using-bootrain
+  - /bootra1n
+  - /bootrain
+  - /using-checkn1x
+  - /using-checknix
+  - /checkn1x
+  - /checkn1x
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md:19]: text and links should be changed to reflect that this is a TV guide

File: docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md [hunk 19]
```diff
@@ -0,0 +1,112 @@
...
+
+::: warning
+
+If you are using a computer with an AMD Ryzen CPU, you will likely run into issues. If you do run into issues, you should use a Mac or a computer with an Intel CPU to follow this guide.
+
+:::
+
+::: warning
+
+A7 devices will likely fail to successfully jailbreak with Odysseyn1x, as a result, you'll need to use an older Odysseyn1x version and quickly unplug and replug your device once you see `Right Before Trigger`. It is highly recommended to follow <router-link to="/installing-chimera">Installing Chimera</router-link> instead.
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md:45]: Should direct to ChimeraTV if this is still applicable to TVs

File: docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md [hunk 20]
```diff
@@ -0,0 +1,112 @@
...
+
+:::
+
+!!!include(./docs/en_US/jailbreak/checkra1n/include/semi-tethered.md)!!!
+
+Odysseyn1x is a live bootable Linux environment that allows you to quickly run checkra1n on a compatible device. checkra1n is capable of jailbreaking millions of iOS devices with A7 to A11 SoC's on firmwares 12.0 to 14.8.1.
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md:51]: TVs don't run iOS

File: docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md [hunk 21]
```diff
@@ -0,0 +1,112 @@
...
+
+!!!include(./docs/en_US/jailbreak/checkra1n/include/odysseyra1n-explanation.md)!!!
+
+On iOS 14, A11 devices do not allow you to use SEP functionality in a jailbroken state. SEP functionality are things such as a passcode, Face ID/Touch ID, and Apple Pay.
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md:55]: TVs don't run iOS and Face ID and Touch ID are not relevant here

File: docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md [hunk 22]
```diff
@@ -0,0 +1,112 @@
...
+
+- A 256MB or greater USB Drive
+    - If you don't have a USB Drive, but do have another form of supported removable bootable hardware (e.g. A CD or DVD), you can use that instead of a USB Drive
+- The latest version of [Odysseyn1x](https://github.com/raspberryenvoie/odysseyn1x/releases)
+    - If you are on an A7 device, you should instead get the v2.12-checkra1n-0.10.1 release of [Odysseyn1x](https://github.com/raspberryenvoie/odysseyn1x/releases/tag/v2.12-checkra1n-0.10.1)
+    - For 64-bit computers, download the "AMD64" version
+        - It doesn't matter if you have an AMD processor or not, "AMD64" is just the name of the architecture
+    - For 32-bit computers, download the "i686" version
+- An Apple TV Siri remote (any generation)
+- If you have an Apple TV 4K, you will also need two cables
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md:67]: colon should be at the end here

Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md:67]: colon should be at the end here

File: docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md [hunk 23]
```diff
@@ -0,0 +1,112 @@
...
+  - A GoldenEye, or Foxlink X892 cable
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md:68]: no comma required here

File: docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md [hunk 24]
```diff
@@ -0,0 +1,112 @@
...
+1. Select the USB drive you would like to boot Odysseyn1x from
+    - This USB drive will be completely erased
+    - Ensure you back up all important data beforehand
+1. Click `Install` and confirm that you are ok erasing the USB drive
+    - Do not remove your USB Drive until the process has completed
+1. Once it has installed, copy the Odysseyn1x iso file that you downloaded onto the USB drive
+
+## Booting Odysseyn1x
+
+1. Reboot, and then go into your BIOS settings and disable Secure Boot, then enter the boot picker and select your USB Drive to boot from
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md:88]: Should be separated into multiple points

File: docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md [hunk 25]
```diff
@@ -0,0 +1,112 @@
...
+    - This is different for every computer
+    - Search your PC or motherboard brand to find out how to get to your BIOS settings if you are unsure
+1. Press enter once you see the Ventoy screen to boot into Odysseyn1x
+
+## Running checkra1n
+
+::: warning
+
+If you have an Apple TV HD and are using a USB-C to UsB-C to do this process, you may run into issues entering into DFU mode
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/using-odysseyn1x-tv.md:97]: should be USB-C not UsB-C

File: docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-linux.md [hunk 26]
```diff
@@ -0,0 +1,64 @@
+---
+lang: en_US
+title: Installing checkra1n for Apple TVs (Linux)
+description: Guide to installing checkra1n for Apple TVs on linux
```
Comment (emiyl) [inline docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-linux.md:4]: should be Apple TV not Apple TVs

File: docs/en_US/jailbreak/installing-backr00m.md [hunk 27]
```diff
@@ -0,0 +1,53 @@
...
+- The latest version of [AltStore](https://altstore.io/)
+- The latest version of [iTunes](https://www.apple.com/itunes/download/win32) if on Windows
+- The latest version of [iCloud]https://secure-appldnld.apple.com/windows/061-91601-20200323-974a39d0-41fc-4761-b571-318b7d9205ed/iCloudSetup.exe) if on Windows
+
+## Installing the application
+
+1. Open AltStore
+1. Plug your Apple TV into your computer
+1. Hold the Shift key if on Windows or the Option key if on Mac and click the AltServer widget in your menu bar
+1. Select your Apple TV from the list, then select the backr00m .ipa
```
Comment (emiyl) [inline docs/en_US/jailbreak/installing-backr00m.md:36]: Should be changed to ```then select the backr00m `.ipa` file```

File: docs/en_US/jailbreak/installing-chimera.md [hunk 28]
```diff
@@ -3,9 +3,6 @@ lang: en_US
 title: Installing Chimera
 description: Guide to installing Chimera
 permalink: /installing-chimera
-redirect_from:
-  - /chimera
-  - /cm
```
Comment (emiyl) [inline docs/en_US/jailbreak/installing-chimera.md:8]: why was this removed?

File: docs/en_US/jailbreak/installing-chimeratv.md [hunk 29]
```diff
@@ -0,0 +1,60 @@
+---
+lang: en_US
+title: Installing ChimeraTV
+description: Guide to installing ChimeraTV
+permalink: /installing-chimeratv
+redirect_from:
+  - /chimera
+  - /cm
```
Comment (emiyl) [inline docs/en_US/jailbreak/installing-chimeratv.md:8]: should be changed to reflect that this is a TV guide

File: docs/en_US/jailbreak/installing-electra.md [hunk 30]
```diff
@@ -3,9 +3,6 @@ lang: en_US
 title: Installing Electra
 description: Guide to installing Electra
 permalink: /installing-electra
-redirect_from:
-  - /electra
-  - /el
```
Comment (emiyl) [inline docs/en_US/jailbreak/installing-electra.md:8]: why was this removed?

File: docs/en_US/jailbreak/installing-electratv.md [hunk 31]
```diff
@@ -0,0 +1,57 @@
+---
+lang: en_US
+title: Installing ElectraTV
+description: Guide to installing electratv
+permalink: /installing-electratv
+redirect_from:
+  - /chimera
+  - /cm
```
Comment (emiyl) [inline docs/en_US/jailbreak/installing-electratv.md:8]: this should be changed as this is not the chimera page

File: docs/en_US/jailbreak/installing-electratv.md [hunk 32]
```diff
@@ -0,0 +1,57 @@
...
+- The latest version of [AltStore](https://altstore.io/)
+- The latest version of [iTunes](https://www.apple.com/itunes/download/win32) if on Windows
+- The latest version of [iCloud]https://secure-appldnld.apple.com/windows/061-91601-20200323-974a39d0-41fc-4761-b571-318b7d9205ed/iCloudSetup.exe) if on Windows
+
+## Installing the application
+
+1. Open AltStore
+1. Plug your Apple TV into your computer
+1. Hold the Shift key if on Windows or the Option key if on Mac and click the AltServer widget in your menu bar
+1. Select your Apple TV from the list, then select the ElectraTV .ipa
```
Comment (emiyl) [inline docs/en_US/jailbreak/installing-electratv.md:40]: should say ```then select the ElectraTV `.ipa` file```

File: docs/en_US/jailbreak/installing-libertv.md [hunk 33]
```diff
@@ -0,0 +1,53 @@
+---
+lang: en_US
+title: Installing LiberTV
+description: Guide to installing LiberTV
+permalink: /installing-LiberTV
```
Comment (emiyl) [inline docs/en_US/jailbreak/installing-libertv.md:5]: permalinks should be lowercase

File: docs/en_US/jailbreak/installing-libertv.md [hunk 34]
```diff
@@ -0,0 +1,53 @@
...
+- The latest version of [AltStore](https://altstore.io/)
+- The latest version of [iTunes](https://www.apple.com/itunes/download/win32) if on Windows
+- The latest version of [iCloud]https://secure-appldnld.apple.com/windows/061-91601-20200323-974a39d0-41fc-4761-b571-318b7d9205ed/iCloudSetup.exe) if on Windows
+
+## Installing the application
+
+1. Open AltStore
+1. Plug your Apple TV into your computer
+1. Hold the Shift key if on Windows or the Option key if on Mac and click the AltServer widget in your menu bar
+1. Select your Apple TV from the list, then select the LiberTV .ipa
```
Comment (emiyl) [inline docs/en_US/jailbreak/installing-libertv.md:36]: should say ```LiberTV `.ipa` file``` at the end instead

**Review by emiyl — CHANGES_REQUESTED** [review]
thank you very much! this is very good. a few changes are required, most quite minor. some are copied from elsewhere in the guide which should be fixed as well at a later date. would do it myself rn but i don't have my laptop with me. one thing i notice that's missing is a package manager usage guide for tvOS, which the pkgman property in the frontmatter references.

**Comment by emiyl** [comment]
thank you very much! this is very good. a few changes are required, most quite minor. some are copied from elsewhere in the guide which should be fixed as well at a later date. would do it myself rn but i don't have my laptop with me. one thing i notice that's missing is a package manager usage guide for tvOS, which the pkgman property in the frontmatter references.

File: docs/en_US/jailbreak/installing-electra.md (hunk 30 above)
Comment (RealKGB) [reply docs/en_US/jailbreak/installing-electra.md:8]: that was a mistake I meant to remove it on installing-electratv.md

**Comment by RealKGB** [comment]
I can go through and fix stuff.

I would write a Using nito.tv guide but I don't have a way to use it atm - my TV 2 is on 5.3 and nito.tv doesn't work there, and my TV 4 is on 15.0.

**Comment by RealKGB** [comment]
Alright everything's fixed I think

**Comment by RealKGB** [comment]
@emiyl

**Comment by emiyl** [comment]
This is mostly done now. I've made the relevant changes to the back end and to AppleDB to facilitate the new changes and it works quite well. One thing we are missing is the "Finding your iOS version" equivalent section. 

image

I don't have an AppleTV so I'm not sure what the text should be, and I can't provide a screenshot either.

A "Using nito.tv" guide should be added as well before merging. Perhaps this info can be sourced from online?

**Comment by RealKGB** [comment]
I can ask converse, I believe they have some ATVs with nito.tv. Although apparently it doesn't work anymore.

I can get the "finding your tvOS version" page done - lemme copy the iOS version one and modify it accordingly.
Should I also write a guide for how to install packages via ssh?

**Comment by RealKGB** [comment]
I cannot find the "finding your iOS version" page... any guidance?

**Comment by emiyl** [comment]
I'm currently away without my laptop but I will review when I get back

**Comment by emiyl** [comment]
> I cannot find the "finding your iOS version" page... any guidance?

`docs/.vuepress/i18n/en_US/chart.js`

```js
      findiOSVersion: {
        title: "Finding your ${osStr} version",
        verStr: {
          old: "Version",
          new: "Software Version",
          tv: "Software Version"
        },
        instructions: {
          ios: "1. Open the Settings application\n1. Navigate to `General\` -> `About` -> `${verStr}`",
          tv: "1. Open the Settings application\n1. Navigate to `General\` -> `About` -> `${verStr}`",
        },
        image: { // For translating, you can take an identical screenshot of these images in the language that you're translating to
          altText: "Finding your ${osStr} version on an ${deviceType}",
          iphone: "/assets/images/find-version.jpg",
          ipad: "/assets/images/find-version-ipad.jpg",
          iphoneOld: "/assets/images/find-version-old.jpg",
          ipadOld: "/assets/images/find-version-ipad-old.jpg",
          tv: "/assets/images/find-version-tv.jpg"
        },
      }
```

I have added apple TV sections for instructions to be added

**Comment by RealKGB** [comment]
Alright, I think I added it - images are there.
I had to split out TV Software and tvOS as they display differently in settings. I'm not sure I did it properly though, can you double-check? Both images are in place.

**Comment by RealKGB** [comment]
Anything left to fix before merge?

**Comment by TheMasterOfMike** [comment]
Rebased into 823cdef, closing
